### ISSUE TITLE
[UPDATE COMPONENT]: 2/3 Versions of Full Width Components

### ISSUE THREAD

**Issue by sarahpipergoldberg** [aclu-national/ptec-design#128]
**Is your component update/variant addition related to a problem? Please describe.**
Some components have full width designs, but the designs aren't extensible, and don't translate to our 2/3 page layout. We'd like to enable the usage of these components on content types that support 2/3 layouts (e.g. campaigns), but the current 2/3 version is not usable/non-existent. 

**Describe the solution you're envisioning**
Create a variant of the existing full width component to work in a 2/3 layout.
NOTE: The goal is to not modify the existing full width component, but instead to use it as the baseline from which to create the 2/3 version. If we need to fully update the full width version, please bring this proposal to the Design Team first, as this will affect other instances of the full width component across the site.

Here are the components that need 2/3 versions:
- Image/Video grid (needs to support both media types)
- Image/Video carousel (using the existing Image Carousel component, not the Playlist component - which we are deprecating)
- Timeline
- Expandable card

**Comment by julesco** [comment]
@sarahpipergoldberg I'd like to align on nomenclature to make sure I understand since these components have gone by different names over the years. Can you link these two items to their page in the design system? For example, there isn't a page called 'playlist' so I'm not 100% sure which media component that's referring to.

- Image/Video grid (needs to support both media types)
- Image/Video carousel (using the existing Image Carousel component, not the Playlist component - which we are deprecating)

**Comment by sarahpipergoldberg** [comment]
hey @julesco ! Sure thing, sorry for the confusion. 

From what I understand, I believe the "Playlist" (which is maybe more of an engineering term) is what is labeled as "[Gallery](https://www.figma.com/file/BVGhVXCShlrXclrCfZIE571U/ACLU-Design-System?type=design&node-id=22348%3A5380&mode=design&t=08kPYLmu91WtrUhn-1)" in Figma. The "[Image Carousel](https://www.figma.com/file/BVGhVXCShlrXclrCfZIE571U/ACLU-Design-System?type=design&node-id=22348%3A5470&mode=design&t=08kPYLmu91WtrUhn-1)" in Figma is what we're hoping to move towards, but to allow for video and images to coexist in this same component (so needs some consideration in terms of aspect ratio and sizing of video vs images). ccing @alisonag to confirm these links and terminologies from an engineering perspective to make sure we're all on the same page!


If you think that it's not possible for image and video to live in the same component at the same time, I think a variant of the same component (one for image and one for video) is fine.

**Comment by julesco** [comment]
@sarahpipergoldberg great, makes sense about retiring gallery and using image carousel and adding video support there. And can you link to what the "Image/Video grid" is referring to, either in figma or on production?

**Comment by sarahpipergoldberg** [comment]
Oh yes, from what I understand, the grid doesn't exist yet (though @alisonag can confirm if this is just a view that we can display on WP?) the idea was to provide two views of the this content - one in carousel, and one in grid, for flexibility for users. So part of this ticket would be to create a grid version that houses images and videos as a variant of the component.

**Comment by alisonag** [comment]
Yes, that [Gallery figma](https://www.figma.com/file/BVGhVXCShlrXclrCfZIE571U/ACLU-Design-System?type=design&node-id=22348%3A5380&mode=design&t=08kPYLmu91WtrUhn-1) link is what we call the Media Playlist in WordPress (for both engineers and editors).

Right now we have the following components that allow you to display collections of images or videos:
- Media Playlist
- Image Grid
- Image Carousel
- Video Carousel

I think it would be better to move the grid vs carousel decision into the component so that editors can play around with the two display options without having to re-add all the media if they change their mind. 

- **Option 1:** Media Collection (with grid or carousel layout options), allows for a mix of video and images. 
  - Requires updated designs for both grid and carousel views to allow mixed content.
  - I originally suggested that we consolidate into one Media component and allow for mixing video (YouTube or self hosted) and images in one grid or carousel because the Media Playlist allows for this, however after reviewing all our live examples I noticed it has never been used with more than one media type; all examples below are either all images or all video. So if from a design perspective you think it's better to only allow one media type per collection (option 2), that's totally fine.
- **Option 2:** Image Collection (with grid or carousel layout options) and Video Collection (with grid or carousel layout options)
  - Requires a new Video Grid design and an updated Video carousel design to allow for other aspect ratios.
  - Image grid and carousel can remain as is

Media Playlist examples: 
- Used on one Campaign: https://www.aclu.org/campaigns-initiatives/defend-your-right-to-learn-join-in-the-fight-against-classroom-censorship
- Used on [10 Landing Pages](https://wp.api.aclu.org/wp-admin/edit.php?s&post_status=all&post_type=landing_page&action=-1&m=0&component_type=shared-media-playlist&filter_action=Filter&paged=1&action2=-1)
- It is available on Flex Page Default template but is not currently in use
